docs/release/qa.md

```diff
@@ -46,28 +46,42 @@ For both:
     ```sh
     docker image pull mathesar/mathesar-caddy:<version_number>
     docker image tag mathesar/mathesar-caddy:<version_number> mathesar/mathesar-caddy:latest
-    
+
     docker image pull mathesar/mathesar:<version_number>
     docker image tag mathesar/mathesar:<version_number> mathesar/mathesar:latest
     ```
 
+### Understanding Database connections
+
+#### Database types
+
+- **Mathesar Database** - The Django database used by Mathesar
+- **Data database** - The user database(s) being explored in Mathesar's UI
+
+#### Connection types
+
+- **Local** - On the host machine
+- **External** - Outside of the host machine
+- **Integrated** - Inside of Mathesar's docker image
+
 ### Tasks
 
 - [ ] Test installing the release from scratch by following our documentation
 - [ ] Test all different installation methods
-    - [ ] Docker compose
-        - [ ] All defaults, local
-        - [ ] Exposed on domain, DB managed by Mathesar
-        - [ ] Exposed on domain, DB preexisting for users, managed DB for Django
-        - [ ] Exposed on domain, DB preexisting for all data
-    - [ ] Docker integrated image
-        - [ ] All defaults, local
-        - [ ] Exposed on domain, DB managed by Mathesar
-        - [ ] Exposed on domain, DB preexisting for users, managed DB for Django
-        - [ ] Exposed on domain, DB preexisting for all data
-    - [ ] Build from scratch
-        - [ ] All defaults, local
-        - [ ] Exposed on domain, DB preexisting for all data
+    - [ ] [Docker compose](https://docs.mathesar.org/latest/administration/install-via-docker-compose/)
+        - [ ] Local using all default settings
+        - [ ] Exposed on domain with different database configurations:
+          - [ ] Integrated Mathesar, Integrated Data
+          - [ ] Integrated Mathesar, External Data
```

Comment:
Hmm, I don't think these points should be listed under the docker compose section. Its quite tedious to test these without making some changes to docker compose file, I suggest listing it as a separate point like before. This method is listed on the first page of [docs](https://docs.mathesar.org/latest/#try-mathesar) so, we need to make sure that it works properly.

Reply:
OOOH, now I understand. This is the piece I was missing. I didn't realize these steps in QA were meant to test running Mathesar _without_ docker compose, using the `docker run -it --name mathesar -p 8000:8000 mathesar/mathesar:latest` command. 

Now I have all of the information I need, I'll revise the PR once more. Thank you!